zhaquirks/tuya/air/ts0601_air_quality.py

```diff
@@ -32,8 +32,7 @@ class TuyaCO2Sensor(CustomDevice):
         # input_clusters=[0, 4, 5, 61184],
         # output_clusters=[25, 10])
         MODELS_INFO: [
-            ("_TZE200_8ygsuhe1", "TS0601"),
-            ("_TZE200_yvx5lh6k", "TS0601"),
+            ("_TZE200_8ygsuhe1", "TS0601")
         ],
```

Comment:
Black also complains here, replace with:
```yaml
        MODELS_INFO: [("_TZE200_8ygsuhe1", "TS0601")],
```

Reply:
This change has nothing to do with "gas leakage sensor".

Please manage it in a separate PR so we can review it properly.